### PR TITLE
remove warnings

### PR DESCRIPTION
**Pull request by GuangweiWang**
as the title.

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -61,7 +61,7 @@ jobs:
       - run: make gmp-bootstrap
       - run: make gtest-bootstrap
       - run: CFLAGS=-Werror make -j$(nproc) all plugin test
-      - run: meson builddir --werror
```
Comment (jrmuizel) [inline .github/workflows/build.yaml:64]: Is this change intentional?

Comment (jrmuizel) [reply .github/workflows/build.yaml:64]: Ah, I didn't realize this was intended for the openh264v2.3.1 branch and not master.

Comment (GuangweiWang) [reply .github/workflows/build.yaml:64]: just for test. no plan to merge

**Comment by GuangweiWang** [comment]
close this PR. use #3594 instead.
